src/shout_subgroup/list_subgroup.py

```diff
@@ -24,6 +56,44 @@ async def list_subgroups(db: Session, telegram_group_chat_id: int) -> list[Type[
     return subgroups
 
 
+async def _handle_list_subgroup_members(update: Update, db: Session, telegram_group_chat_id: int, subgroup_name: str):
```

Comment:
Focus: New helper method for listing subgroup members. Responsible for sending the reply messages